### PR TITLE
Server endpoint exclude duplicates

### PR DESCRIPTION
**Pull request by mpage21**
After this is merged, I'll run the code to create the all associations files

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,9 @@ The PRSKB website offers users the ability to calculate polygenic risk scores ac
 Website Link: [https://prs.byu.edu](https://prs.byu.edu)
 
 ### Calculate
-The Calculate page of the PRSKB website allows the user to calculate polygenic risk scores for multiple samples across more than 2100 trait/study combinations. Using an input VCF or inputted rsIDs and genotypes, reference genome, super population of the samples, default biological sex, p-value cutoff, and selected studies polygenic risk scores can be calculated for the inputted samples. Results can be downloaded as a tsv or json file. 
+The Calculate page of the PRSKB website allows the user to calculate polygenic risk scores for multiple samples across more than 2100 trait/study combinations. Using an input VCF or inputted rsIDs and genotypes, reference genome, super population of the samples, p-value cutoff, and selected studies polygenic risk scores can be calculated for the inputted samples. Results can be downloaded as a tsv or json file.
```
Comment (lizvance) [inline README.md:12]: comma after "and selected studies"

**Review by lizvance — COMMENTED** [review]
Just the one comma that needs to be added! Nice work :)

File: static/js/controllers/associations.controller.js [hunk 2]
```diff
@@ -320,49 +320,98 @@ async function separateStudies(associations, traitData, refGen, sex) {
...
-                    if (!('traitsWithDuplicateSnps' in studyIDsToMetaData[association.studyID])) {
-                        studyIDsToMetaData[association.studyID]['traitsWithDuplicateSnps'] = [association.trait]
+                    if (!('traitsWithExcludedSnps' in studyIDsToMetaData[association.studyID])) {
+                        studyIDsToMetaData[association.studyID]['traitsWithExcludedSnps'] = [association.trait]
                     }
-                    else if (!(studyIDsToMetaData[association.studyID]['traitsWithDuplicateSnps'].includes(association.trait))) {
-                        studyIDsToMetaData[association.studyID]['traitsWithDuplicateSnps'].push(association.trait)
+                    else if (!(studyIDsToMetaData[association.studyID]['traitsWithExcludedSnps'].includes(association.trait))) {
+                        studyIDsToMetaData[association.studyID]['traitsWithExcludedSnps'].push(association.trait)
+                        // TODO we will need to put in the documentation that the symbol we use for duplicated snps should indicate that we have excluded those duplicated snps
```
Comment (MattCloward) [inline static/js/controllers/associations.controller.js:357]: Have you completed this TODO?

File: static/js/controllers/associations.controller.js [hunk 3]
```diff
@@ -320,49 +320,98 @@ async function separateStudies(associations, traitData, refGen, sex) {
...
+                    potentialDupliSnpsInfo.push([association.snp, association.trait, association.studyID])
+                }
             }
         }
         // else add the pos->trait->studyID 
         else if (association.studyID in studyIDsToMetaData){
-            AssociationsBySnp[association.snp] = {
-                pos: association[refGen],
-                traits: {}
+            //todo check if this is the right logic
```
Comment (MattCloward) [inline static/js/controllers/associations.controller.js:398]: Have you done this TODO?

**Review by MattCloward — CHANGES_REQUESTED** [review]
There are two TODOs that need to be addressed and removed. Other than that, this looks good!

**Pull request merged**